File: mario.py
from cs50 import get_int

# while height is not between 1 and 8, get int, other wise break the loop
while True:
    height = get_int('Height: ')
    if height < 1 or height > 8:
        True
        print('Height between 1 and 8')

    else:
        break
# for each row in range of height
for row in range(height):
    # print spaces, signs (row + 1) times, then 2 spaces and then another signs (row + 1) times.
    space = height - row - 1
    print(' ' * (space), end='')
    print('#' * (row + 1), end='')
    print('  ', end='')
    print('#' * (row + 1))

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // create the variable "height" without any value
    int height;

    do
    {
        // Prompt user for a height
        height = get_int("Height: ");
    }
    // if the number is not between 1 and 8, do again first step
    while (height < 1 || height > 8);

    // row 0 is height 1, so row must be less than height
    for (int row = 0; row < height; row++)
    {
        // print space first in nested loop row
        for (int space = 0; space < height - row - 1; space++)
        {
            printf(" ");
        }
        // column is nested in row, so everytime row is increased, cloumn initialize to 0, thats why column must be less or equal than row
        for (int col = 0; col <= row; col++)
        {
            printf("#");
        }
        printf("\n");
    }


}